src/main/java/org/killbill/billing/entitlement/api/BaseEntitlementWithAddOnsSpecifier.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.entitlement.api;
+
+import java.util.UUID;
+
+import org.joda.time.LocalDate;
+
+public interface BaseEntitlementWithAddOnsSpecifier {
+
+    /**
+     * @return the bundle Id
+     */
+    public UUID getBundleId();
```

Comment:
@javier-gomez-github I am not sure exactly why @pierre made that comment. He is at at a conference tight now, so unclear if he will have time to comment. I would keep it the way it is for now, and we can always revisit.